models/services.ts

```diff
@@ -1,43 +1,49 @@
 export type Status = 'Draft' | 'Published'
 
-export interface Service {
+export type ResourceStatus =
```

Comment:
Switching to types over interfaces because we can't index interfaces with strings (i.e. `Interface[stringField]`) without adding [an index signature](https://www.typescriptlang.org/docs/handbook/2/objects.html#index-signatures).